src/rust/dbv2/migrations/2024-01-09-160715_price_to_nominal/up.sql

```diff
@@ -0,0 +1,35 @@
+-- Your SQL goes here
+CREATE FUNCTION integer_price_to_quote_indivisible_subunits(market_id numeric, price numeric) RETURNS NUMERIC IMMUTABLE AS $$
+    SELECT ($2 / ((tick_size / POW(10::numeric,quote.decimals)) / (lot_size / POW(10,COALESCE(base.decimals, 0)))))
+    FROM market_registration_events
+    LEFT JOIN aggregator.coins base
+    ON base_account_address = base."address" AND base_module_name = base.module AND base_struct_name = base.struct
+    INNER JOIN aggregator.coins quote
+    ON quote_account_address = quote."address" AND quote_module_name = quote.module AND quote_struct_name = quote.struct
+    WHERE market_id = $1;
+$$ LANGUAGE sql;
+
+CREATE FUNCTION quote_indivisible_subunits_to_integer_price(market_id numeric, price numeric) RETURNS NUMERIC IMMUTABLE AS $$
+    SELECT $2 * ((tick_size / POW(10::numeric,quote.decimals)) / (lot_size / POW(10,COALESCE(base.decimals, 0))))
```

Comment:
How about similarly consolidating the algebra?

```sql
($2 * lot_size * POW(10::numeric, quote.decimals - COALESCE(base.decimals, 0)) / tick_size)
```

Reply:
This just needs to be 1/(the other function).